Animations/Zoom/Left.js

```diff
@@ -26,5 +28,5 @@ var ZoomLeft = {
     'transform': 'scale(.1) translateX(-20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomLeft', ZoomLeft);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomLeft', ZoomLeft);
```

Comment:
This does not need to have `'ZoomLeft'` passed into it since we have defined a `className` property in `ZoomLeft`.